src/components-examples/cdk/stepper/cdk-custom-stepper-without-form/example-custom-stepper.css

```diff
@@ -1,7 +1,8 @@
 .example-container {
-  border: 1px solid black;
+  border: 1px solid;
   padding: 10px;
   margin: 10px;
+  color: inherit;
```

Comment:
I think all of these `color: inherit` uses are redundant. Doesn't `color` default to `inherit`?

Reply:
Fixed, kept color: inherit for nav buttons since those were shown in black for dark mode.